tests/unittests/sources/test_azure.py

```diff
@@ -3799,6 +3802,171 @@ def test_no_pps(self):
         # Verify dmesg reported via KVP.
         assert len(self.mock_report_dmesg_to_kvp.mock_calls) == 1
 
+    def test_no_pps_gpa(self):
```

Comment:
Please add a docstring on this test and the next one which describes what is being tested.

Reply:
be sure to link to https://github.com/azure/guestproxyagent

Reply:
done